Userland/Libraries/LibGUI/GroupBox.cpp

```diff
@@ -24,6 +24,14 @@ GroupBox::~GroupBox()
 {
 }
 
+Margins GroupBox::content_margins() const
+{
+    return { (!m_title.is_empty() ? font().glyph_height() + 1 /*room for the focus rect*/ : 2),
+        2,
+        2,
+        2 };
```

Comment:
These magic numbers are already gone in my working branch, where I make a lot of changes to GroupBox, but are just here as a stop-gap solution, for now.